src/plots/plots.js

```diff
@@ -2828,6 +2828,15 @@ plots.doCalcdata = function(gd, traces) {
         );
     }
 
+    // clear relinked cmin/cmax values in shared axes to start aggregation from scratch
+    for(var k in fullLayout._colorAxes) {
```

Comment:
Do we need to turn this to
```js
for(var k in (fullLayout._colorAxes || [])) {
```
?

Reply:
`_colorAxes` is initialised during `Plots.supplyDefaults`:

https://github.com/plotly/plotly.js/blob/513491e6a8f20e247881609eefa74ffdb6c7c319/src/plots/plots.js#L390

which should always be called before `Plots.doCalcdata`, so adding a fallback here would be pretty much useless.